### PR TITLE
Added Secret and Webhook for GitHub

### PR DESCRIPTION
**Pull request by Arman084**

### CODE REVIEW

File: .openshift/templates/ninja-board-back-end.yml [hunk 1]
```diff
@@ -21,6 +21,14 @@ objects:
...
         name: ${DOCKER_IMAGE_REPOSITORY}
       name: latest
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+    name: ${SECRET_REF}
+  data:
+    WebHookSecretKey: bmluamEtYmFjaw==
```
Comment (sabre1041) [inline .openshift/templates/ninja-board-back-end.yml:31]: Should this be parameterized? Also, you can use the `stringData` field of a secret to have OpenShift/Kubernetes automatically performing a base64 encode

Comment (Arman084) [reply .openshift/templates/ninja-board-back-end.yml:31]: Yeah, I'll make the suggested changes

Comment (Arman084) [reply .openshift/templates/ninja-board-back-end.yml]: @sabre1041 I just updated the template, let me know if there are more changes to perform. Thanks

**Comment by Arman084** [comment]
I'm not sure why the other changes were considered for this PR, let me just create a brand new PR and submit it.

**Pull request closed without merging**